### ISSUE TITLE
Cant get to work on Windows

### ISSUE THREAD

**Issue by eohne** [leferrad/OCReract.jl#26]
Hi, 
I have installed tesseract (version 5.4.0) and can extract text using CMD commands.  Unfortunately, the julia package  (v1.3.1) does not work for me. I am running Julia version 1.9.0 on Windows 11.

```julia
  [916415d5] Images v0.26.1
  [c9880795] OCReract v1.3.1
```

I have also run the tests and got the following output:
```julia
Testing Running tests...
┌ Error: Error ocurred while running Tesseract! Base.IOError("could not spawn `tesseract 'C:\\Users\\elias\\.julia\\packages\\OCReract\\Ax3XS\\/test/files/testocr.png' 'C:\\Users\\elias\\AppData\\Local\\Temp\\jl_M1QJGPKbgD/res.txt' --oem 1 --psm 3`: no such file or directory (ENOENT)", -4058)
└ @ OCReract C:\Users\elias\.julia\packages\OCReract\Ax3XS\src\tesseract.jl:137
RunTesseract: Test Failed at C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\test_tesseract.jl:33
  Expression: res == true
   Evaluated: false == true

Stacktrace:
 [1] macro expansion
   @ C:\Users\elias\AppData\Local\Programs\Julia-1.9.0\share\julia\stdlib\v1.9\Test\src\Test.jl:478 [inlined]
 [2] test_run_tesseract()
   @ Main C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\test_tesseract.jl:33
RunTesseract: Error During Test at C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\test_tesseract.jl:123
  Got exception outside of a @test
  SystemError: opening file "C:\\Users\\elias\\AppData\\Local\\Temp\\jl_M1QJGPKbgD/res.txt": No such file or directory
  Stacktrace:
    [1] systemerror(p::String, errno::Int32; extrainfo::Nothing)
      @ Base .\error.jl:176
    [2] #systemerror#82
      @ .\error.jl:175 [inlined]
    [3] systemerror
      @ .\error.jl:175 [inlined]
    [4] open(fname::String; lock::Bool, read::Bool, write::Nothing, create::Nothing, truncate::Nothing, append::Nothing)
      @ Base .\iostream.jl:293
    [5] open
      @ .\iostream.jl:275 [inlined]
    [6] open(fname::String, mode::String; lock::Bool)
      @ Base .\iostream.jl:356
    [7] open(fname::String, mode::String)
      @ Base .\iostream.jl:355
    [8] open(::var"#3#4", ::String, ::Vararg{String}; kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ Base .\io.jl:393
    [9] open
      @ .\io.jl:392 [inlined]
   [10] test_run_tesseract()
      @ Main C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\test_tesseract.jl:36
   [11] macro expansion
      @ C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\test_tesseract.jl:125 [inlined]
   [12] macro expansion
      @ C:\Users\elias\AppData\Local\Programs\Julia-1.9.0\share\julia\stdlib\v1.9\Test\src\Test.jl:1498 [inlined]
   [13] top-level scope
      @ C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\test_tesseract.jl:125
   [14] include(fname::String)
      @ Base.MainInclude .\client.jl:478
   [15] macro expansion
      @ C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\runtests.jl:4 [inlined]
   [16] macro expansion
      @ C:\Users\elias\AppData\Local\Programs\Julia-1.9.0\share\julia\stdlib\v1.9\Test\src\Test.jl:1498 [inlined]
   [17] top-level scope
      @ C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\runtests.jl:4
   [18] include(fname::String)
      @ Base.MainInclude .\client.jl:478
   [19] top-level scope
      @ none:6
   [20] eval
      @ .\boot.jl:370 [inlined]
   [21] exec_options(opts::Base.JLOptions)
      @ Base .\client.jl:280
   [22] _start()
      @ Base .\client.jl:522
Test Summary:  | Fail  Error  Total  Time
OCReractTest   |    1      1      2  5.7s
  RunTesseract |    1      1      2  3.1s
ERROR: LoadError: Some tests did not pass: 0 passed, 1 failed, 1 errored, 0 broken.
in expression starting at C:\Users\elias\.julia\packages\OCReract\Ax3XS\test\runtests.jl:3
ERROR: Package OCReract errored during testing
```

Any help would be very much appreciated!

**Comment by leferrad** [comment]
Hi @eohne ! Sorry for the late reply - it seems you don't have `tesseract` installed (or executable from terminal maybe), as your error is related to that:

>  could not spawn `tesseract 'C:\\Users\\elias\\.julia\\packages\\OCReract\\Ax3XS\\/test/files/testocr.png' 'C:\\Users\\elias\\AppData\\Local\\Temp\\jl_M1QJGPKbgD/res.txt' --oem 1 --psm 3`: no such file or directory (ENOENT)

Have you installed tesseract before as indicated in the README? Notice that #20 was not solved yet, so this package assumes you already installed tesseract by yourself.

**Comment by eohne** [comment]
Hi @leferrad 
Thanks for getting back to me. Yes I did! 
I installed version 5.4 before the Julia package and also added it to PATH. 
I am running Windows btw. 

Best
Elias

**Comment by leferrad** [comment]
Can you open a terminal (e.g. Git Bash) and run `tesseract -h` to see if the binary could be recognized by Julia? If so, ensure the image file and the output folder exist to avoid the `no such file or directory` being related just to that issue.

**Comment by eohne** [comment]
Hi @leferrad `tesseract -h` works fine. I have also used tesseract from the terminal successfully. My "workaround" was to store the images to disk, then extract the text using the terminal, and finally to import the text files back into Julia. Which was tedious but I didn't have a lot of images so it worked fine.

**I have just reinstalled OCReract and for some reason, it works now and installs/precompiles without a problem.  Sorry about this the Issue can be closed.**
